src/pages/index.js

```diff
@@ -118,75 +167,129 @@ function getCardElement(data) {
   return cardElement;
 }
 
-function handleEscape(evt) {
-  if (evt.key === "Escape") {
-    const openedModal = document.querySelector(".modal_is-opened");
-    if (openedModal) {
-      closeModal(openedModal);
-    }
-  }
-}
+// Delete form submit handler
+deleteModalForm.addEventListener("submit", function handleDeleteSubmit(evt) {
+  evt.preventDefault();
+  deleteBtnText(deleteModalForm, true, "delete", "deleting...");
```

Comment:
* According to the task, you need to change submit buttons text (`Saving…`) while waiting for the server response. And the user should see it.

And `Deleting...` for the delete button


You should pass to the `deleteBtnText()` function  the **save button** as the 1st argument instead of the `form`